src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/gadgets/MultiTool.java

```diff
@@ -86,12 +94,33 @@ protected ItemUseHandler getItemUseHandler() {
                     }
                 }
             } else {
+                final String LORE_PREFIX = ChatColors.color("&8\u21E8 &7Mode: ");
+                final Pattern REGEX = Pattern.compile(ChatColors.color("(&c&o)?" + LORE_PREFIX) + "(.+)");
```

Comment:
Global these too pls, right now we're compiling this regex on every tool use. We only need to compile it once